packages/search-input/src/SearchResultsMenu/SearchResultsMenu.style.ts

```diff
@@ -13,9 +13,11 @@ export const searchResultsMenuStyles = css`
 export const searchResultsMenuThemeStyles: Record<Theme, string> = {
   [Theme.Light]: css`
     background-color: ${palette.white};
+    border: 1px solid ${palette.gray.light2};
   `,
   [Theme.Dark]: css`
     background-color: ${palette.gray.dark3};
+    border: 1px solid rgba(255, 255, 255, 0);
```

Comment:
palette.white?

Reply:
This is for darkMode so i left it transparent but i could make it mongo black